unity-client/Assets/Scripts/MainScripts/DCL/Components/UUIDComponent/InteractionHoverCanvas/InteractionHoverCanvasController.cs

```diff
@@ -24,8 +25,7 @@ public class InteractionHoverCanvasController : MonoBehaviour
 
     void Awake()
     {
-        Environment.i.SetInteractionHoverCanvasController(this);
```

Comment:
Had to remove the `InteractionHoverCanvasController` from environment for now. `Environment` is not supposed to deal with `Monobehaviour` and this flow in its current state will give race condition issues.